e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -56,6 +56,37 @@ describe('Desktop notifications', () => {
         });
     });
 
+    it('MM-T482 Desktop Notifications - (at) here not rec\'d when logged off', () => {
+        cy.apiCreateUser({}).then(({user}) => {
+            cy.apiAddUserToTeam(testTeam.id, user.id);
+            cy.apiLogin(user);
+
+            // # Ensure notifications are set up to fire a desktop notification if are mentioned.
+            changeDesktopNotificationSettingsAs('#desktopNotificationMentions');
+            stubNotificationAs('withoutNotification', 'granted');
+
+            cy.apiGetChannelByName(testTeam.name, 'Off-Topic').then((res) => {
+                const channel = res.body;
+
+                // # Logout the user.
+                cy.apiLogout();
```

Comment:
The test case mentioned to wait for a couple minutes (but maybe few seconds is enough), so I suggests
```suggestion
                cy.apiLogout().wait(TIMEOUTS.TEN_SEC);
```